### PR TITLE
[Jason Lim] Implement UpdateFoodIntake

### LINKED ISSUE #75: Issue: Multiple similar food intake items are repetitively listed in a day
Multiple similar food items can be added in a day and it is reflected as individual items in the food intake list repetitively. Should we aggregate them as a unique item identified by a multiplier at the back?

### LINKED ISSUE #90: As a user, I want to be able to edit the macronutrients of a previously added food intake
So that the food intake list will be up-to-date and get a more accurate diet plan report.

### PR DESCRIPTION
**Pull request by jasaaanlim**
Added the update food intake functionality to update the macronutrient values for a previously added FoodIntake since they are not linked to the Food list.

Currently, only the macronutrient is updated and may be updated after further discussions - to see if the name and date should be updated.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/UpdateFoodIntakeCommand.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.food.Food;
+import seedu.address.model.food.FoodIntake;
+
+/**
+ * Updates a particular FoodIntake item in the FoodIntakeList.
+ */
+public class UpdateFoodIntakeCommand extends Command {
+    public static final String COMMAND_WORD = "updatefoodintake";
```
Comment (ssoonwee) [inline src/main/java/seedu/address/logic/commands/UpdateFoodIntakeCommand.java]: I feel the command should be rephrased to foodintakeupdate for consistency with the rest

Comment (jasaaanlim) [reply src/main/java/seedu/address/logic/commands/UpdateFoodIntakeCommand.java]: Good point! Will update this.

Comment (jasaaanlim) [reply src/main/java/seedu/address/logic/commands/UpdateFoodIntakeCommand.java]: Updated command name.

**Review by ssoonwee — APPROVED** [review]
Looks good!

**Comment by jasaaanlim** [comment]
#90

**Pull request merged**